bin/s2nd.c

```diff
@@ -372,8 +372,15 @@ int handle_connection(int fd, struct s2n_config *config, struct conn_settings se
         echo(conn, fd);
     }
 
+    /* TODO: Some integration tests fail with this change. This is a work around until we fix those. */
```

Comment:
"with this change": I'd specify what change, so people don't have to git-blame to figure out the context.